app/views/submitted_content/_hyperlink.html.erb

```diff
@@ -26,10 +28,120 @@
             <input type="radio" id='chk_links' name='chk_links' value="<%= index %>"/>
           <% end %>
         <% end %>
-              <a href="<%= link %>" target="_blank"><%= link %></a>
-              <br/>
+        <% if local_assigns[:submission_check] %>
+          <a href="<%= link %>" class="review_link"><%= link %></a>
+        <% else %>
+          <a href="<%= link %>" target="_blank"><%= link %></a>
+        <% end %>
+        <br/>
       <% end %>
       <% } %>
     </ul>
   </div>
+  <% if local_assigns[:submission_check] %>
+    <script type="text/javascript">
+      function ISODateString(date){
+        function pad(num){return num<10? '0'+num: num;}
+        return date.getUTCFullYear()+'-'
+                  + pad(date.getUTCMonth()+1) + '-'
+                  + pad(date.getUTCDate()) + ' '
+                  + pad(date.getUTCHours()) + ':'
+                  + pad(date.getUTCMinutes()) + ':'
+                  + pad(date.getUTCSeconds());
+      }
+      $(document).ready(function(){	    
+        var map_id = <%= @map.id %>;
+        var round = <%= @current_round %>;
+        var time_out_interval = 500; // in milliseconds
+        var linksToUpdate = {};
+        var windowsToClose = new Array();
+        $("#url a.review_link").on("contextmenu", function() {
```

Comment:
Disable right-clicking so that users cannot circumvent the "left" click listener (supported on all browsers) that starts the review tracking.